Bump urllib3

maarcingebala opened the pull request:
Bump `urllib3` and `elasticsearch` (the latter was required to bump the first one).

### Pull Request Checklist

1. [ ] Privileged views and APIs are guarded by proper permission checks.
1. [ ] All visible strings are translated with proper context.
1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
1. [ ] Database queries are optimized and the number of queries is constant.
1. [ ] Database migration files are up to date.
1. [ ] The changes are tested.
1. [ ] The code is documented (docstrings, project documentation).
1. [ ] GraphQL schema and type definitions are up to date.
1. [ ] Changes are mentioned in the changelog.

jxltom commented:
- The ```requirement.txt``` and ```requirements_dev.txt``` don't need to be edited manually, pipenv could do that work for you by ```pipenv lock -r > requirements.txt``` and ```pipenv lock -r -d > requirements_dev.txt```.
- It looks like you are not using the latest pipenv, and I checked that lockfile has some extra unnecessary markers which is caused by a bug in the previous version of pipenv and fixed in the latest version. Do you mind upgrade your pipenv and do the lock again by ```pipenv lock```? Thanks.

maarcingebala commented:
@jxltom Thanks for your feedback! I've updated Pipenv, changed the Pipfile as you suggested and regenerated requirements files. For the `elasticsearch` package I left a specific version, as I remember that automatically bumping it caused some issues in the code once and it had to be fixed manually.

jxltom commented:
Cool, now this looks great to me!
